index.js

```diff
@@ -80,6 +80,32 @@ const parseInput = async input => {
 	return input;
 };
 
+const killChecker = (input, options) => {
+	if (typeof options.verify !== 'undefined' && options.verify === false) {
+		return;
+	}
+
+	const seconds = options.verifyTimeout || 2;
+	let checks = seconds * 10;
+
+	function timeout() {
+		return new Promise(resolve => setTimeout(resolve));
+	}
+
+	async function checker() {
+		checks--;
+
+		const exists = await processExists(input);
+
+		if (exists && checks > 0) {
+			timeout();
```

Comment:
What's the purpose of this? Since it's not awaited.

Reply:
Can you add a test that ensures it takes the time given in `verifyTimeout`? You can use https://github.com/sindresorhus/delay#delayrangeminimum-maximum-options for this, to give it a slight threshold.